test/testhelpers/loggerSpy.ts

```diff
@@ -0,0 +1,22 @@
+import { Logger } from 'winston';
+
+export class LoggerSpy {
+  private mockWriter: jest.SpyInstance<boolean, [any, string, ((error: Error) => void)?]>;
+
+  constructor(logger: Logger) {
+    this.mockWriter = jest.spyOn(logger.transports[0], 'write');
+  }
```

Comment:
I'm wondering if it would be useful to have an optional parameter that defaults to 0 for indicating which transport you want to spy on, just in case we need to spy on a different transport later. But, that can probably wait until the situation actually arises.

Reply:
That's definitely a good option for the future, but I agree that for now we probably don't need it.